kitsune/users/jinja2/users/email/contributor.ltxt

```diff
@@ -66,7 +66,7 @@ L10N Contributor - Help to translate Firefox Help content
 to your language:
 {% endtrans %}
 
-{{ add_utm('https://support.mozilla.org/en-US/kb/localize-firefox-help', 'get-involved', source='new-contributor') }}
```

Comment:
I know that localize-firefox-help redirects to localize-mozilla-support but have you talked with the folks watching google analytics about this change? I suggest that you remove this change from this PR since it's unrelated anyway and if it's OK with the GA folks pull request seperatelly.